pkg/storage/tableland/store.go

```diff
@@ -33,22 +33,10 @@ var (
 			fleet text not null,
 			rig_attributes_value text not null,
 			position integer not null,
-			cid text not null,
+			path text not null,
```

Comment:
Since we're moving to the new schema design, this was changed to a more appropriate column name (we don't store any cid in there, just the path within a root cid).